### ISSUE TITLE
another type

### ISSUE THREAD

**Issue by 1C0D** [Fevol/obsidian-typings#38]
for files in the explorer :
    interface View {
        fileItems: fileItem[]
    }

    interface fileItem{
        "collapsible": boolean,
        "collapsed" : boolean,
        "el": HTMLElement,
        "selfEl": HTMLElement,
        "coverEl": HTMLElement,
        "collapseEl": HTMLElement,
        "innerEl": HTMLElement,
        "childrenEl": HTMLElement,
        "info": any,
        "view": View,
        "file": TFile | TFolder,
        "tagEl": HTMLElement,
        "parent": fileItem
    }

![image](https://github.com/Fevol/obsidian-typings/assets/63018905/274ac164-8f69-4e2a-8bd8-d08b90c211ea)

**Comment by 1C0D** [comment]
I did this
```
    interface fileItem{
        "collapsible": boolean,
        "collapsed" : boolean,
        "el": HTMLElement,
        "selfEl": HTMLElement,
        "coverEl": HTMLElement,
        "collapseEl": HTMLElement,
        "innerEl": HTMLElement,
        "childrenEl": HTMLElement,
        "info": any,
        "view": View,
        "file": TFile | TFolder,
        "tagEl": HTMLElement,
        "parent": fileItem
    }
    ```
    it was enough for my needs

**Comment by Fevol** [comment]
Thanks for the info! This is now released with v1.0.4
